### DISCUSSION TITLE
/bin/node: No such file or directory

### DISCUSSION THREAD

**Discussion by andystevenson** [johnlindquist/kit#864] [Errors]
Cannot run commands from the terminal as they are looking for /bin/node rather than ~/.knode/bin/node which I think is what they are supposed to be looking at.

Also ~/.kenv/bin is not in the PATH environment variable which I think. it should be as part of the install?

**Comment by johnlindquist** [comment]
1. /bin/node

Can you show me the contents of the command you're trying to run? For example, the contents of my testing-run-from-terminal located at: `~/.kenv/bin/testing-run-from-terminal`

```bash
#!/bin/sh

NODE_NO_WARNINGS=1 \
KENV=$(cd "$(dirname ${BASH_SOURCE[0]})"/.. &> /dev/null && pwd) \
/Users/johnlindquist/.knode/bin/node \
/Users/johnlindquist/.kit/run/terminal.js \
/Users/johnlindquist/.kenv/scripts/testing-run-from-terminal.js \
"$@"
```

2. PATH

I didn't feel comfortable automatically adjusting a user's PATH without their permission. You can add the path by using this:

![CleanShot 2022-08-11 at 08 07 12@2x](https://user-images.githubusercontent.com/36073/184152522-429eeeae-1aa3-494a-9e4f-8cf082fb6ae4.png)

**Comment by andystevenson** [comment]
Hi John,

Thanks for the reply. Missed Step 2. Add kenv bin to $PATH

I downloaded my-ip from the community link and this is what I got.

```
#!/bin/sh

NODE_NO_WARNINGS=1 \
KENV=$(cd "$(dirname ${BASH_SOURCE[0]})"/.. &> /dev/null && pwd) \
/bin/node \
/Users/andystevenson/.kit/run/terminal.js \
/Users/andystevenson/.kenv/scripts/my-ip.js \
"$@"

```

I also downloaded word-game from the community site and got the same

```
➜ word-game
/Users/andystevenson/.kenv/bin/word-game: line 3: /bin/node: No such file or directory

```

**Reply by andystevenson** [reply]
Frustratingly still got this. I deleted and reinstalled and still getting the same issue. Is there a way I can debug why it is happening in my environment?

**Reply by johnlindquist** [reply]
Sorry, I'll be back online in a couple hours to help debug.

We're starting a Discord server if you'd prefer to chat in real-time

https://discord.gg/qcWXkZJC
